kafka-client/src/main/java/dev/responsive/kafka/store/ResponsivePartitionedStore.java

```diff
@@ -210,25 +214,17 @@ public byte[] get(final Bytes key) {
       return result.isTombstone ? null : result.value;
     }
 
-    return client.get(name.cassandraName(), partition, key);
+    return client.get(name.cassandraName(), partitioner.partition(partition, key), key);
   }
 
   @Override
   public KeyValueIterator<Bytes, byte[]> range(final Bytes from, final Bytes to) {
-    return new LocalRemoteKvIterator<>(
```

Comment:
why do we need to do this? with the NO_SUBPARTITIONS partitioner hard-coded this still works right?

Reply:
yes, but the next PR changes that :)